queries/snakemake/highlights.scm

```diff
@@ -13,7 +13,7 @@
     name: _ @keyword))
 
 ; Subordinate directives (eg. input, output)
-(_
+((_)
   body: (_
     (directive
       name: _ @label)))
```

Comment:
This changes from finding `body` field that are child of any nodes, to finding `body` fields that have any node before it, making a lot of captures. I don't think this is correct

Reply:
The old parser broke when interspersing comments in rules. This is fixed.
However, when a comment is inserted immediately after opening a rule, the comment node is a child of `rule_definition`, not of `rule_body`. This is the same structure as when inserting a comment immediately after opening a vanilla Python class. 

The old capture (with `_` instead of `(_)`) does not work there.
![queries](https://github.com/user-attachments/assets/aad13930-8fe0-4d12-8e24-64919983980c)

Reply:
Then can't the query only be
```suggestion
body: (_
  (directive
    name: _ @label))
```

Reply:
That seems to work. I did not know this was valid syntax.